Modules/Salary/Resources/views/employee/index.blade.php

```diff
@@ -12,26 +12,31 @@
         @endif
         <div class="mt-4 card">
             <div class="card-header pb-lg-5 fz-28">
-                <div class="mt-4 ml-5">Employee Salary</div>
+                <div class="mt-4 ml-5">Employee Salary ( <i class="fa fa-rupee"></i>&nbsp;)</div>
             </div>
             <div class="card-body">
                 <div class="d-flex justify-content-between mx-5 align-items-end">
                     <h1>{{$employee->name}}</h1>
                     @if(optional($employee->user()->withTrashed())->first()->avatar)
+                    <span class="content tooltip-wrapper"  data-html="true" data-toggle="tooltip"  title="{{ $employee->name }}">
                         <img src="{{ $employee->user()->withTrashed()->first()->avatar }}" class="w-100 h-100 rounded-circle">
+                    </span>
                     @endif
                 </div>
-
                 <hr class='bg-dark mx-4 pb-0.5'>
                 <br>
-                <div class="form-group">
-                    <label class="leading-none ml-5" for="grossSalary">{{ __('Monthly Gross Salary') }}</label>
-                    <input type="number" step ="0.01" name="grossSalary" id="grossSalary" placeholder="Enter Monthly Gross Salary" value="{{ optional($employee->employeeSalary->first())->monthly_gross_salary }}" required>
+                <div class="input-group col-md-9 fz-24 ml-3"><b>Current Monthly Gross Salary:&nbsp;</b>{{ optional($employee->employeeSalary->first())->monthly_gross_salary }}</div>
```

Comment:
This should be last no? Because as employees will keep having some more salary entries, we need to pick the last one (latest one)
```suggestion
                <div class="input-group col-md-9 fz-24 ml-3"><b>Current Monthly Gross Salary:&nbsp;</b>{{ optional($employee->employeeSalary->last())->monthly_gross_salary }}</div>
```

Reply:
yes, right now we are using createorupdate method to store the data in database so that's why I had used first but last() is more suitable according to the requirements and use cases.